public/app/features/explore/state/explorePane.test.ts

```diff
@@ -102,8 +102,8 @@ describe('refreshExplore', () => {
           .givenThunk(refreshExplore)
           .whenThunkIsDispatched(exploreId);
 
-        const initializeExplore = dispatchedActions[1] as PayloadAction<InitializeExplorePayload>;
-        const { type, payload } = initializeExplore;
+        const initializeExplore = dispatchedActions.find(action => action.type === initializeExploreAction.type);
```

Comment:
The actions tests here are not ideal as they test what is dispatched which is a bit of an internal implementation detail. Ideally at some moment the tests would just dispatch action and check final state after the action.